.github/workflows/ci_uploads.yml

```
@@ -0,0 +1,39 @@
+        run: |
+          FILE="./internal/com.espressif.idf.uploads/cmakeed/CMakeEd-1.24.1.zip"
+          FILE_NAME=$(basename "$FILE")
+          BUCKET_NAME="dl.espressif.com"
+          REMOTE_PATH="dl/cmakeed/updates/"
+
+          if [ -f "$FILE" ]; then
+            echo "Uploading $FILE_NAME to S3..."
+            aws s3 cp "$FILE" "s3://$BUCKET_NAME/$REMOTE_PATH$FILE_NAME" --acl public-read
+          else
+            echo "File $FILE not found, skipping..."
+          fi
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)